packages/odyssey-storybook/.storybook/preview-head.html

```diff
@@ -27,4 +27,23 @@
   .sbdocs .sbdocs-a {
     color: #1662dd;
   }
+
+  .sbdocs .sb-dont {
```

Comment:
We didn't end up needing this decorator for the `Banner` docs, so I can either remove it here or clean up the styles for merge.